src/Juvix/FrontendContextualise/InfixPrecedence/Transform.hs

```diff
@@ -0,0 +1,481 @@
+{-# LANGUAGE LiberalTypeSynonyms #-}
+
```

Comment:
Can we add a top-level comment to this file with a pseudo-syntax description of what the transform does?

Reply:
Sure I will give the changes in the format defined in FrontendDesugar

Reply:
See the Extend.hs file for this